row.py

```diff
@@ -167,8 +168,8 @@ def convert_pdf_to_jpg_bytes(pdf_as_bytes):
 
     try:
         images = convert_from_bytes(pdf_as_bytes, dpi)
-    except (TypeError, PDFInfoNotInstalledError, PDFPageCountError, PDFSyntaxError) as error:
         logging.error(error)
+    except (TypeError, PDFInfoNotInstalledError, PDFPageCountError, PDFSyntaxError, DecompressionBombError) as error:
```

Comment:
Is this for the cv2 error with huge files? Should we skip files exceeding a certain size and log them?

Reply:
Yeah, the logs show this as some sort of attack. Catching the error I think is more the python way.